src/common/main.lua

```diff
@@ -276,7 +285,7 @@ function ant.init()
 		local antProcessIdToReassign = msg.Tags["Process-Id"]
 
 		-- send the release message to the provided IO Process Id
-		ao.send({
+		utils.Send(msg, {
```

Comment:
i vaguely recall there being an issue with using utils.Send when it was first introduced, can you review other PRs and make sure we're not causing any regressions here?

Reply:
Yeah i cant use utils in notices - causes a circular dependancy :( 

Apart from that, there was a potential issue with not having `Target`, because internally it uses msg.reply, but if you specify a target it uses it, so thats fine.